v3/opt/datadog/datadog-mesos-master.service

```diff
@@ -0,0 +1,32 @@
+# https://raw.githubusercontent.com/DataDog/docker-dd-agent/master/dd-agent.service
+# make sure the etcd /ddapikey is set!
+[Unit]
+Description=Datadog Agent for Mesos master
+After=docker.service
+
+[Service]
+# CONTROL_ELB should be set in etcd
+# It should be run on all mesos master nodes
+# Look at adobeplatform/docker-dd-agent-mesos for what ports those 3 should be running on
+User=core
+Restart=always
+TimeoutStartSec=0
+ExecStartPre=/usr/bin/docker pull index.docker.io/adobeplatform/docker-dd-agent-mesos-master:latest
+ExecStartPre=-/usr/bin/docker kill dd-agent-mesos-master
+ExecStartPre=-/usr/bin/docker rm -f dd-agent-mesos-master
+ExecStart=/usr/bin/bash -c \
+"if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && sudo /usr/bin/docker run --name dd-agent-mesos-master -h `hostname` \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v /proc/:/host/proc/:ro \
+-v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+-e API_KEY=`etcdctl get /datadog/config/api-key` \
+-e HOST_IP=`hostname -i` \
+-e MARATHON_USERNAME=`etcdctl get /marathon/config/username` \
```

Comment:
@chr0n1x : This came out of the conversation on Thursday, we wanted to move the marathon checks from being just on a single marathon master (the single-instance `datadog-control` unit) to run on each master, so the name of the container is really now like really something like `datadog-mesos-master-marathon`, functionally. BUT... that meant a repo change, plus we're maybe going to end up consolidating datadog containers/repos MAYBE eventually, so didn't feel like going through the jumps of renaming/removing repos at this juncture. Makes sense?